src/main/resources/META-INF/plexus/components.xml

```diff
@@ -0,0 +1,18 @@
+<component-set>
+    <components>
+        <component>
+            <role>com.gradle.develocity.agent.maven.api.DevelocityListener</role>
+            <role-hint>common-custom-user-data-develocity-listener</role-hint>
+            <implementation>com.gradle.CommonCustomUserDataDevelocityListener</implementation>
+            <description>Captures common custom user data in Maven build scans</description>
```

Comment:
Let's do amin-step forward and change build scans to Build Scan (capitalized, singular).

Reply:
Done as part of latest changes.